modules/candidate_list/js/columnFormatter.js

```diff
@@ -45,32 +45,9 @@ function formatColumn(column, cell, rowData) {
                 );
         }
     }
-    
-    // For those candidates who had their scans done
-    // make the 'Scan Done' value in the column a link
-    // to the imaging browser with the filter's PSCID
-    // set to the PSCID of the candidate
-    if (column === 'Scan Done' && cell === 'Y') {
-        return React.createElement(
-            "td",
-            { className: "scanDoneLink" },
-            React.createElement(
-                "a",
-                { 
-                  onClick: function(e) {
-                      loris.loadFilteredMenuClickHandler(
-                          'imaging_browser', {"pscid" : rowData[2]}
-                      )(e);
-                  }
-                },
-                cell
-            )
-        );
-    }
-
```

Comment:
Because someone changed the compiled file and not the jsx file. This pull request was based of another one i did thinking it would have been merged by now